src/pages/memberspage/modal.tsx

```
@@ -0,0 +1,91 @@
+import { Role } from "../../store/members";
+import { useTranslation } from "react-i18next";
+
+interface ModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  member: Member;
+}
+
+const Modal = ({ isOpen, onClose, member }: ModalProps) => {
+  if (!isOpen) return null;
+  const { i18n } = useTranslation();
```

ESLint

[error] React Hook "useTranslation" is called conditionally. React Hooks must be called in the exact same order in every component render.

(react-hooks/rules-of-hooks)